docs/content/users/quickstart.md

```diff
@@ -54,7 +54,7 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ```bash
     mkdir my-cakephp-site && cd my-cakephp-site
     ddev config --project-type=cakephp --docroot=webroot
-    ddev composer create --prefer-dist cakephp/app:~5.0
+    ddev composer create --prefer-dist --no-interaction cakephp/app:~5.0
     ddev cake
```

Comment:
```suggestion
```
i think the `ddev cake` line can be dropped since it is missing an actual command doing something. at the moment it results in:  ```$> ddev cake
No command provided. Choose one of the available commands.```

the cake php docs also dont use the cake command setting up a new project: https://book.cakephp.org/5/en/installation.html#installation-using-ddev

Reply:
Yes, it makes sense to remove that line, thanks.